native/hdfs/src/object_store/hdfs.rs

```diff
@@ -88,19 +88,33 @@ impl HadoopFileSystem {
 
     fn read_range(range: &Range<u64>, file: &HdfsFile) -> Result<Bytes> {
         let to_read = (range.end - range.start) as usize;
+        let mut total_read = 0u64;
```

Comment:
wondering would be that better to have `total_read` as usize?

Reply:
changed to usize